modules/datagovuk/service.tf

```
@@ -0,0 +1,164 @@
+locals {
+  formatted_allowed_ips = [for ip in try(var.secrets["allowed_ip_addresses"], []) : "\"${split("/", ip)[0]}\"/${split("/", ip)[1]}"]
+  template_values = merge(
+    { # some defaults
+      origin_port = 443
+      minimum_tls_version = "1.2"
+      ssl_ciphers = "ECDHE-RSA-AES256-GCM-SHA384"
+      basic_authentication = null
+      probe_dns_only = false
```

You definitely want `dummy = true` on your probe config, otherwise you'll probably take out the origin service with probes 😄
```suggestion
      probe_dns_only = true
```

---

I copied the defaults from [assets](https://github.com/alphagov/govuk-fastly/blob/a0dd5a92009a77c752495b9dcdd047148c1bbb48/modules/assets/service.tf#L8). It gets [overwritten to `true` in the variable](https://github.com/alphagov/govuk-fastly-secrets/commit/38aecba093c3be9f8f05334eabcf831bd5b482d2#diff-08c80ed11902eefef09739d41c91408238bb8b5e7be7cc1e5db933b7c8de65c3R393).

Better to be on the safe side 😅 and I probably should have left it out.